Core/StitchCoreSDK/Sources/StitchCoreSDK/Internal/Net/StitchAppRoutes.swift

```diff
@@ -23,27 +24,27 @@ public protocol StitchAuthRoutes {
      * The route on the server for getting a new access token.
      */
     var sessionRoute: String { get }
-
+    
```

Comment:
[style] all the extra tabs

Reply:
[meta q] mostly a question for Jason, but is our build process in a stable enough state that we can do the add linter build phases ticket (STITCH-1597)? It was previously in limbo because SPM would overwrite build phases when generating the xcodeproj but now that all of our Xcode projects are checked into the repo and cocoapods doesn't overwrite them, adding the linter phases should be trivial (aside from actually getting our code to pass the linter)

Reply:
Yes.

Reply:
fixed